[2024-08-23T11:44:42.066+0000] {processor.py:161} INFO - Started process (PID=46) to work on /opt/airflow/dags/dag_script.py
[2024-08-23T11:44:42.067+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-23T11:44:42.070+0000] {logging_mixin.py:188} INFO - [2024-08-23T11:44:42.069+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-23T11:44:42.091+0000] {logging_mixin.py:188} INFO - [2024-08-23T11:44:42.087+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-23T11:44:42.093+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-23T11:44:42.115+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.056 seconds
